interface/src/app/gene/gene-network.tpl.html

```diff
@@ -3,7 +3,7 @@ <h1 class="page-header">
     Gene Network
   </h1>
   <div class="well">
-    <gene-search-form></gene-search-form>
+    <gene-search-panel organism="organism"></gene-search-panel>
```

Comment:
Interesting. I just noticed that the template corresponding to this directive is named `gene-search-form.tpl.html` rather than `geneSearchForm.tpl.html`. I didn't know that either style could work! The tutorials I used when learning Angular all had this `camelCase.tpl.html` template file name style mixed with `<camel-case>...` directive name style.

Since you have the hyphenated style of naming everywhere, I guess it works! Wondering if we should standardize on one approach vs. the other. What are your thoughts, @dongbohu and @rzelayafavila?